docs/concepts/05_Advanced/terms-and-quote.md

```diff
@@ -0,0 +1,129 @@
+---
+id: terms-and-quote
+title: Terms and Quote
+---
+
+The Claimer may request and / or the Attester may send the Terms of the attestation, i.e., the requirements set by the participants (the Claimer and the Attester) for the conditions of the attestation.
+
+These Terms are defined and agreed upon before the attestation is created. This part of the process requires interaction and communication between both parties. This communication can be done independently, e.g. in person, via messaging, on social media etc., or via the KILT software development kit (SDK).
+
+## Sending Terms using the KILT SDK
+
+Both “request terms” and “submit terms” are part of the messaging system: the message is sent as “request terms” and received as “submit terms”.
+
+The interaction is as follows:
+
+- The Claimer creates a partial `Claim` (optionally) and sends a message to the Attester, requesting the Terms
+- An Attester creates a `Terms` object and sends it, as part of a "submit terms" message, back to the Claimer.
+- The Claimer receives the message, checks the Terms and, if all is in order, agrees to them
+
+## How to create a Terms object
+
+When creating a Terms object, at least one of the following must be set: Legitimations, Quote or Delegations.
+
+- **Legitimations**: Legitimations are Credentials, issued to the Attester the Claimer wants to interact with, showing that the Attester has the authority or legitimacy to attest the claim requested. This is a way of establishing trust between the participants.
+
+- **Quote**: Details on how to create a Quote are outlined here. In brief, the Quote could include any or all of the following:
+
+  - The time frame the claim will be completed in
+  - The public address of the Attester completing the attestation
+  - The hash of the corresponding CType
+  - A breakdown of all the (net and gross) costs, the currency quoted, and any taxes associated with the attestation.
+  - A link to the terms and conditions of the attestation.
+
+- **Delegations**: An Attester may include the right to attest to a specific claim from a top-down trust authority e.g. a business giving the right to their employee to attest the claim on their behalf. This is a delegation. If the Attester is delegating their attestation rights to another entity, this should be stated clearly at this point. See here for details on how to create a delegation.
+
+## How to create a Terms object using the KILT SDK
```

Comment:
I feel we have crossed the line of what can be here and what could go in the cookbook. Does **Not even JSON** also applies to the advanced concepts? 😛

Reply:
Puh, yeah I am not so sure either.
Do we only show the function calls in the cookbook, or also the resulting JSON objects?

Reply:
It's saying "using the KILT SDK", so for me it feels natural to put it in the cookbook. In the concepts (although an advanced one), I would focus on what terms and quotes are, and what they are used for.